src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -4888,6 +4875,14 @@ class BodyVisitor : public CommonVisitor<BodyVisitor> {
             tmp = nullptr;
             tmp_vec.clear();
             visit_stmt(*x.m_body[i]);
+            if (!global_init.empty()) {
+                for (auto t: global_init) {
+                    if (t) {
+                        items.push_back(al, t);
+                        global_init.erase(t);
+                    }
+                }
+            }
```

Comment:
I think this can be simplified to

```suggestion
            for (auto t: global_init) {
                if (t) {
                    items.push_back(al, t);
                }
            }
            global_init.n = 0;
```